App/src/main/java/com/yagni/model/MarketRepo.java

```diff
@@ -0,0 +1,152 @@
+            "<br> <br>Top customers by dollar amount:<br>1.  " + firstCustomer + " ---> $" + firstAmt + "<br>2. " + secondCustomer + " ---> $" +  secAmt + "<br>3. " + thirdCustomer + " ---> $" + thirdAmt + 
+            "</html>", "Weekly Market Report", JOptionPane.INFORMATION_MESSAGE);
+
+        }
+
+            catch(SQLException e) {
+                System.out.println(e);
+                System.out.println("Oops an error occurred!");
+            } 
+        }
+    public void monthReport(DbConnection linkDb){
+            
+        try{
+            Statement state = linkDb.getConnection().createStatement();
+    
+            //=============== FOR MONTHLY REPORT ===============//
+            ResultSet result = state.executeQuery("SELECT product_id, sum(product_quanity) AS 'sum' FROM yagni_inv_db.purchase_history WHERE order_date > DATE_ADD(curdate(),INTERVAL -1 MONTH) AND order_date <= curdate() GROUP BY product_id ORDER BY sum(product_quanity) DESC LIMIT 3;");
+            result.next();    
+            firstProduct = result.getString(1);     //1st "Most popular order"
+            result.next();
+
+            secondProduct= result.getString(1);     //2nd "most popular order"            
+            result.next();
+            
+            thirdProduct = result.getString(1);     //3rd "most popular order"    
+            result = state.executeQuery("SELECT (SELECT sum(product_quanity) FROM purchase_history) * product.Sale_cost AS 'totalSpent', customer_email FROM product RIGHT JOIN purchase_history ON product.Sale_cost = purchase_history.product_quanity LEFT JOIN  yagni_inv_db.customer_information ON purchase_history.hashed_email = customer_information.hashed_email WHERE  order_date > date_add(curdate(), INTERVAL - 1 MONTH) AND order_date <= curdate() group by customer_email ORDER BY totalSpent DESC LIMIT 3;");
+            result.next();
+
+            firstCustomer= result.getString(2);     //1st "most popular customer"
+            firstAmt = result.getString(1);
+    
+            result.next();
+            secondCustomer= result.getString(2);    //2nd "most popular customer"
+            secAmt = result.getString(1);
+    
+            result.next();
+            thirdCustomer= result.getString(2);    //3rd "most popular customer"    
+            thirdAmt = result.getString(1);       
+
+            //creating a pop-up that displays a message w/ variables created 
+            JOptionPane.showMessageDialog(null,"<html>Most popular orders:<br>1. " + firstProduct + "<br>2. " + secondProduct + "<br>3. " + thirdProduct + 
+            "<br> <br>Top customers by dollar amount:<br>1.  " + firstCustomer + " ---> $" + firstAmt + "<br>2. " + secondCustomer + " ---> $" +  secAmt + "<br>3. " + thirdCustomer + " ---> $" + thirdAmt + 
+            "</html>", "Weekly Market Report", JOptionPane.INFORMATION_MESSAGE);
```

Comment:
Since this message prints the Monthly report this string should read Monthly Market Report. Can we make this change before merging? @Jesusxx-240

Reply:
Yup I'm on it.